src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,32 @@
+package codeu.chat.common;
+
+import java.util.logging.Level;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import codeu.chat.util.Uuid;
+
+public class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
```

Comment:
could we make this a Uuid instead of a String and that way avoid the error check in the constructor?